flare/ase/otf.py

```diff
@@ -164,8 +170,17 @@ def update_positions(self, new_pos):
             curr_velocities = self.atoms.get_velocities()
             self.atoms.set_velocities(curr_velocities * vel_fac)
 
-    def update_gp(self, train_atoms, dft_frcs):
+    def update_temperature(self):
+        self.KE = self.atoms.get_kinetic_energy()
+        self.temperature = self.atoms.get_temperature()
+
+        # Convert velocities to Angstrom / ps.
+        self.velocities = self.atoms.get_velocities() * units.fs * 1e3
 
+    def run_dft(self):
+        super().run_dft()
+
```

Comment:
We can remove this `run_dft` function, if it is not defined, it will automatically use the `run_dft` function in the parent class

Reply:
Thanks for catching this, I've removed the method in 9b13052